mount/losetup_linux.go

```diff
@@ -94,6 +94,7 @@ func setupLoopDev(backingFile, loopDev string, param LoopParams) (*os.File, erro
 
 	// 2. Set FD
 	if _, _, err = ioctl(loop.Fd(), unix.LOOP_SET_FD, back.Fd()); err != nil {
+		loop.Close()
```

Comment:
Can we just use `defer loop.Close()` before the `// 2. Set FD` line?

Reply:
This function should only close fd if it fails.
https://github.com/containerd/containerd/blob/746cef0bc2afe8b8c4592fec3df966138bdcedff/mount/losetup_linux.go#L129-L136